src/compiler/utilities.ts

```diff
@@ -491,6 +491,10 @@ namespace ts {
             return getTokenPosOfNode(node.jsDoc![0], sourceFile);
         }
 
+        if (node.kind === SyntaxKind.JsxText) {
+            return skipTrivia((sourceFile || getSourceFileOfNode(node)).text, node.pos, /*stopAfterLineBreak*/ false, /*stopAtComments*/ true);
+        }
+

```

Comment:
The important bit is that the trvia detection will not stop at comments rather than absording them

Reply:
Isn't this the same as the `isJSDocComment` branch just above? Can they be combined?

Reply:
Confirmed it has the same behavior, and they can be combined - @andrewbranch I can't push to this branch, nor make a suggestion which lives that far out of the diff to give you a one click fix. Can you edit it? Otherwise I'll make a new PR